public/js/actions/breakpoints.js

```diff
@@ -84,7 +84,8 @@ function _removeOrDisableBreakpoint(location, isDisabled) {
     if (bp.loading) {
       // TODO(jwl): make this wait until the breakpoint is saved if it
       // is still loading
-      throw new Error("attempt to remove unsaved breakpoint");
+      // throw new Error("attempt to remove unsaved breakpoint");
+      return;
```

Comment:
I'd still like to error here. This is a bug and we should suppress the error. Note that the current debugger acts this way too; we didn't implement proper add/remove ordering. The error doesn't break the UI; it just stops this redux dispatch cycle, but the state is still in tact and the UI keeps working.

I'd rather properly fix this. This is what the `waitUntil` middleware is for; because we don't have a good abstraction for these kinds of things, it allows you to wait on other actions in a primitive way. Here, if it's still loading, it should dispatch a `waitUntil` request can check when the bp isn't loading anymore, and then call `removeBreakpoint` again. I can do that, so remove this change and I'll implement a proper fix.